libbpfgo.go

```diff
@@ -96,6 +98,15 @@ func BPFMapTypeIsSupported(mapType MapType) (bool, error) {
 	return supportedC == 1, nil
 }
 
+func BPFFuncTypeIsSupported(progType BPFProgType, funcId helpers.BPFFunc) (bool, error) {
```

Comment:
For clarity, I suggest renaming to `BPFHelperIsSupported`

Reply:
+1